dtm0/dtm0.h

```diff
@@ -91,6 +91,72 @@
  * @{
  */
 
+#include "fid/fid.h"            /* m0_fid */
+#include "fid/fid_xc.h"         /* m0_fid_xc */
+#include "lib/buf.h"            /* m0_bufs */
+#include "lib/buf_xc.h"         /* m0_bufs_xc */
+#include "lib/types.h"          /* uint64_t */
+#include "xcode/xcode.h"        /* M0_XCA_RECORD */
+
+
+struct m0_dtx0_id {
+	struct m0_fid dti_originator_sdev_fid;
+	uint64_t      dti_timestamp;    /* XXX fix the type */
```

Comment:
In the m0_dtx0_id_cmp(), the timestamp is compared first, and then the fid.

So, I think we can move the timestamp to the head as the first element.

Reply:
Yes, i will update this in dld branch(PR).